spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -60,6 +60,7 @@ def __init__(self):
 step1 = Paramreg(step='1', type='imseg', algo='centermassrot', rot_method='pcahog')
 step2 = Paramreg(step='2', type='seg', algo='bsplinesyn', metric='MeanSquares', iter='3', smooth='1', slicewise='0')
 paramregmulti = ParamregMultiStep([step0, step1, step2])
+step_rootlets = Paramreg(step='1', metric='CC', iter='6x6x3', shrink='8x4x2', smooth='0x0x0', slicewise='0', deformation='0x0x1', gradStep='0.1')
```

Comment:
consider adding a flag `-param-rootlets` in the CLI

Reply:
Since landmark registration is called "step #0", and we perform rootlets segmentation immediately after, it's like we're essentially inserting a new step in-between step 0 and step 1. 

So, maybe we should make it a requirement that if `-lrootlets` is specified, we only apply the rootlets-based reg if the user specifies `-param step=1,type=rootlets,[...]`. That way, `-param` can be re-used instead of `-param-rootlet`, and it also makes it 100% explicit to the user that the rootlets-based registration will be performed immediately after step=0.

(Right now, it's kind of ambiguous, and the user has no control over when or how the rootlets reg will be inserted into the substeps.)